commit 6963338ca586bef3370e189f13785b4b6b63bbdb
Author: Karel Zak <kzak@redhat.com>
Date:   Thu Apr 5 15:59:59 2018 +0200

    dmesg: fix raw output
    
    The commit 5f538ac40c0d215b6c9a881effe4c3b214366715 has introduced
    regression ("goto" to the wrong place, so timestamps in raw mode is
    not printed at all).
    
    Addresses: https://github.com/karelzak/util-linux/issues/614
    Signed-off-by: Karel Zak <kzak@redhat.com>

diff --git a/sys-utils/dmesg.c b/sys-utils/dmesg.c
index 22c381c1d..ba4e22528 100644
--- a/sys-utils/dmesg.c
+++ b/sys-utils/dmesg.c
@@ -906,7 +906,7 @@ static void print_record(struct dmesg_control *ctl,
 				       LOG_MAKEPRI(rec->facility, rec->level),
 				       (long) rec->tv.tv_sec,
 				       (long) rec->tv.tv_usec);
-		goto mesg_output;
+		goto full_output;
 	}
 
 	/* Store decode information (facility & priority level) in a buffer */
@@ -1008,7 +1008,6 @@ full_output:
 			color_disable();
 	}
 
-mesg_output:
 	/*
 	 * A kernel message may contain several lines of output, separated
 	 * by '\n'.  If the timestamp and decode outputs are forced then each
